source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -6537,6 +6537,26 @@ export class ActionSet extends CardElement {
         return super.findDOMNodeOwner(node);
     }
 
+    getElementById(id: string): CardElement | undefined {
+        let result = super.getElementById(id);
+
+        if (!result) {
+            for (let i = 0; i < this.getActionCount(); i++) {
+                const action = this.getActionAt(i);
+
+                if (action instanceof ShowCardAction) {
+                    result = action.card.getElementById(id);
+
+                    if (result) {
+                        break;
+                    }
+                }
+            }
+        }
+
+        return result;
+    }
```

Comment:
this code is repeated twice - could you please refactor?
see if it can be placed in the parent class of the both classes where it's used?:)